public/javascripts/API.js

```diff
@@ -5,7 +5,7 @@
 var api = {
   login : function(params){
     $.ajax({
-      url : "http://" + window.location.host + "/api/auth",
+      url : "http://" + window.location.host + "/api/v1/login",
```

Comment:
Right. Makes sense. 

Sent from my iPhone

> On May 27, 2016, at 11:09 PM, Ryan Stanley <notifications@github.com> wrote:
> 
> In public/javascripts/API.js:
> 
> > @@ -5,7 +5,7 @@
> >  var api = {
> >    login : function(params){
> >      $.ajax({
> > -      url : "http://" + window.location.host + "/api/auth",
> > +      url : "http://" + window.location.host + "/api/v1/login",
> I believe I put that there for people testing with docker's IP vs localhost vs deployed
> …
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>